dashboard_server.py

```
@@ -0,0 +1,530 @@
+                    for sf in summary_files:
+                        if detailed_ts in os.path.basename(sf):
+                            selected_summary = sf
+                            break
+                    # Otherwise pick the latest by timestamp suffix after `_summary_`
+                    if not selected_summary:
+                        try:
+                            selected_summary = sorted(
+                                summary_files,
+                                key=lambda p: os.path.basename(p).split('_summary_')[1]
+                            )[-1]
+                        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)